app/models/favorite.rb

```diff
@@ -19,4 +19,8 @@ def pets_by_id
     end
   end
 
+  def destroy_pet(pet)
+    @contents.delete(pet)
```

Comment:
This is beautiful!

Reply:
Kind of just the opposite of the add pets. I tried to copy the Pets.destroy from the pet class, but it didn't work as I hoped.

Reply:
Yeah, and it looks like it only deletes the id from favorites contents, I'm glad it's not deleting the actual record.